docs/ops/deployment/docker.md

```diff
@@ -395,13 +395,13 @@ The next chapters show examples of configuration files to run Flink.
     flink run -d -c ${JOB_CLASS_NAME} /job.jar
     ```
 
-  * or copy the JAR to the *Flink Master* container and submit the job using the [CLI](..//cli.html) from there, for example:
+  * or copy the JAR to the *JobManager* container and submit the job using the [CLI](..//cli.html) from there, for example:
 
     ```sh
     JOB_CLASS_NAME="com.job.ClassName"
-    MASTER_CONTAINER=$(docker ps --filter name=jobmanager --format={% raw %}{{.ID}}{% endraw %}))
-    docker cp path/to/jar "${MASTER_CONTAINER}":/job.jar
-    docker exec -t -i "${MASTER_CONTAINER}" flink run -d -c ${JOB_CLASS_NAME} /job.jar
+    JM_CONTAINER=$(docker ps --filter name=jobmanager --format={% raw %}{{.ID}}{% endraw %}))
```

Comment:
It is quite unfortunate that the abbreviation `JM` is now ambiguous.

Reply:
Could do `JOBMANAGER_CONTAINER`. WDYT?

Reply:
I consider the `JobMaster` an internal component that users rarely see, though. Or should see.

Reply:
I'm fine with using JM here; this was more of a general remark how this could be an issue in future discussions.